src/index.js

```diff
@@ -45,7 +45,7 @@ const edgeCorePlugins = {
   foxExchange: makeFoxExchangePlugin,
   godex: makeGodexPlugin,
   shapeshift: makeShapeshiftPlugin,
-  sideshift: makeSideShiftPlugin,
+  sideShift: makeSideShiftPlugin,
```

Comment:
I appreciate the eye for detail to casing, but after reviewing other variable names in our codebase we should revert this. The consensus for names seems to be to use `sideshift`. (like `shapeshift`)